tamr_toolbox/utils/recipe.py

```diff
@@ -0,0 +1,35 @@
+"""Tasks related to project recipe"""
+import logging
+
+from tamr_unify_client.mastering.project import MasteringProject
+
+from tamr_toolbox.models.project_type import ProjectType
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_dedup_recipe_id(project: MasteringProject) -> int:
+    """
+    Retrieves dedup recipe ID of a mastering project.
+    To be used to run custom recipe operations.
+
+    Args:
+        project: a mastering project
+
+    Returns: dedup recipe id of the project.
+
+    """
+    # Retrieve project steps using unified dataset
```

Comment:
perhaps we could add a check here to make sure the type is consistent and raise a TypeError:
```suggestion

    #check project type
    not_mastering_error = (
        f"{project.name} is not a mastering project."
    )
    if project.type != ProjectType.DEDUP.value:
        LOGGER.error(not_mastering_error)
        raise Exception(not_mastering_error)
    # Retrieve project steps using unified dataset
```